src/ngx_http_lua_util.c

```diff
@@ -4266,4 +4266,25 @@ ngx_http_lua_set_sa_restart(ngx_log_t *log)
 #endif
 
 
+size_t
+ngx_http_lua_safe_header_filed_len(u_char *str, size_t len)
```

Comment:
Better make this an inline function defined in `util.h` instead to avoid C function call overhead.